qcodes/instrument_drivers/Keysight/keysight_e4980a.py

```diff
@@ -256,6 +266,14 @@ def measure_impedance(self) -> MeasurementPair:
     def measurement(self) -> MeasurementPair:
         return self._measurement()
 
+    def _set_range(self, val: str) -> None:
+        """
+        Sets range for impedance measurements and updates
+        `imp_autorange_enabled` cache value.
+        """
```

Comment:
Seeing the code of this method (and also seeing that it's private), this docsrting seems redundant :)
```suggestion
```